evals/benchmark/stresscli/locust/aistress.py

```diff
@@ -169,6 +173,22 @@ def bench_main(self):
                                         complete_response += content
                                 except json.JSONDecodeError:
                                     continue
+                        elif self.environment.parsed_options.bench_target in ["faqgenfixed", "faqgenbench"]:
```

Comment:
where is the input/output length statistics

Reply:
Input: [num_token_input_prompt](https://github.com/opea-project/GenAIEval/commit/fb8577d93913c71085b6fcdcc9b497cada720d50#diff-86d2bee7782ede400308e9dad1dcfdcaa545957f2d747b4db875c1ef289b2f31L19)
Output: [complete_response](https://github.com/opea-project/GenAIEval/blob/33641c63a323b46441671e36ea7d5fcb7e3f8524/evals/benchmark/stresscli/locust/aistress.py#L189) -> [response_string](https://github.com/opea-project/GenAIEval/blob/33641c63a323b46441671e36ea7d5fcb7e3f8524/evals/benchmark/stresscli/locust/aistress.py#L205C1-L206C66) -> [num_token_output](https://github.com/opea-project/GenAIEval/blob/33641c63a323b46441671e36ea7d5fcb7e3f8524/evals/benchmark/stresscli/locust/tokenresponse.py#L28C1-L30C6)